pkg/configs/configs.go

```
@@ -71,6 +71,10 @@ func MergeRemoteConfigs(ctx context.Context, state *state.State) (map[string]env
 }
 
 func ListRemoteConfigs(ctx context.Context, state *state.State) ([]api.Config, error) {
+	if state.RemoteClient == nil {
```

Why not return an error in this case? Is this expected?